Sources/DCoreKit/Model/NftDataType.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+public struct NftDataType: Codable {
+    public let type: DataType
+    public let unique: Bool
+    public let modifiable: ModifiableBy
+    public let name: String?
+
+    public enum ModifiableBy: String, Codable, Equatable {
+        case
+        nobody,
+        issuer,
+        owner,
+        both
+
+        var ordinal: Int {
+            switch self {
+            case .nobody: return 0
+            case .issuer: return 1
+            case .owner: return 2
+            case .both: return 3
+            }
+        }
+    }
+
+    public enum DataType: String, Codable {
```

Comment:
NftDataType.DataType is little ambiguous/confusing, can u suggest different name for DataType.